packages/bubble-dev/start-preset/src/build.ts

```diff
@@ -87,36 +88,54 @@ export const buildReactNative = async (dir: string): Promise<StartPlugin<{}, {}>
 
 export const buildNode = async (dir: string): Promise<StartPlugin<{}, {}>> => {
   const { babelConfigNodeBuild } = await import('@bubble-dev/babel-config')
-
-  return sequence(
-    env({ BABEL_ENV: 'production' }),
-    find([
+  const { default: globby } = await import('globby')
```

Comment:
I needed to rewrite node build to be more similar to react native one, so that in case of two files with the same name and different extensions it would choose the one with `.node.(js|jsx|ts|tsx)` extension.